common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -206,6 +252,17 @@ def author_view(self, context):
 
         return fragment
 
+    def studio_view(self, _context):
+        """
+        Return the studio view.
+        """
+        fragment = Fragment(
+            self.system.render_template(self.mako_template, self.get_context())
+        )
+        add_webpack_to_fragment(fragment, 'ConditionalBlockStudio')
+        shim_xmodule_js(fragment, self.studio_js_module_name)
+        return fragment
```

Comment:
@s0b0lev What was missing was an override of this method (which is defined in `MakoTemplateBlockBase`) to add the webpack fragment. If you cherry-pick this commit and run `make studio-static` and `make studio-restart`, it should work now.